router/aspects/request_timeout_handler.go

```diff
@@ -0,0 +1,41 @@
+package aspects
+
+import (
+	"github.com/julienschmidt/httprouter"
+	"github.com/prebid/prebid-server/config"
+	"net/http"
+	"strconv"
+)
+
+func QueuedRequestTimeout(f httprouter.Handle, reqTimeoutHeaders config.RequestTimeoutHeaders) httprouter.Handle {
+
+	return func(w http.ResponseWriter, r *http.Request, params httprouter.Params) {
+
+		reqTimeInQueue := r.Header.Get(reqTimeoutHeaders.RequestTimeInQueue)
+		reqTimeout := r.Header.Get(reqTimeoutHeaders.RequestTimeoutInQueue)
+
+		//If request timeout headers are not specified - process request as usual
+		if reqTimeInQueue == "" || reqTimeout == "" {
+			f(w, r, params)
+			return
+		}
+
+		reqTimeFloat, reqTimeFloatErr := strconv.ParseFloat(reqTimeInQueue, 64)
+		reqTimeoutFloat, reqTimeoutFloatErr := strconv.ParseFloat(reqTimeout, 64)
+
+		//Return HTTP 400 if request timeout headers are incorrect (wrong format)
+		if reqTimeFloatErr != nil || reqTimeoutFloatErr != nil {
```

Comment:
I think we should either have some documentation on this behavior, or send some descriptive text along with the error to indicate where the issue is. I can see a host being confused where a misconfigured header results in all the calls failing.